store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -77,8 +77,9 @@
 
   /**
    * Sets the output path of the writer builder
+   *
```

Comment:
Can avoid the space changes in multiple places if not intentional.

Reply:
I think it's better for formatting the code comments/annotation